Directory.Build.props

```diff
@@ -18,12 +18,14 @@
   </PropertyGroup>
 
   <!-- NO_MOBILE can be passed in via an environment variable or a build property to disable all mobile targets -->
-  <PropertyGroup Condition="'$(NO_MOBILE)' == 'true'">
+  <PropertyGroup Condition="'$(NO_MOBILE)' == ''">
```

Comment:
We probably don't want this checked into source control

Reply:
We might need a second `sln` after all. Unles we find another way to avoid this.

Can we have a symlink? `Sentry.NoMobile.sln` is a symlink to `Sentry.sln`? So you can add the checks back on the msbuild files based on sln file name?